### PR TITLE
[GH-603] adjusted readme structure

### LINKED ISSUE #603: Restructure and clean readme, add docs folder with more specific documents

### PR DESCRIPTION
**Pull request by thepiwo**
- I removed the HTTP-API docs as they may be outdated and should not be used currently anyways
- added some more developer docs

@DanielaIvanova @gspasov do you have some more things you want to add to the developer docs?

resolves #603

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,362 +1,71 @@
...
 
-  Where *block_hash* is the bech32 encoded block header hash
+### Elixir interactive api-calls
+- `Miner.resume()` to start the miner
+- `Miner.suspend() ` to stop the miner
 
-- To get an account balance:
 
-  `GET localhost:4000/balance/{account}`
+- `Chain.top_block()` to add get the top block of the current chain
```
Comment (DanielaIvanova) [inline README.md]: Supposed to be: to get the top block of the current chain

Comment (DanielaIvanova) [reply README.md]: Maybe will be good to be add `Miner.mine_sync_block_to_chain` as well. What do you think?

Comment (thepiwo) [reply README.md]: yes, will adjust and add this

**Comment by DanielaIvanova** [comment]
Looks good!

**Pull request merged**